### ISSUE TITLE
Accessibility Features are Missing

### ISSUE THREAD

**Issue by StormProductionsMusic** [builtbybel/ReadySunValley#5]
Hi there,

Very great idea on making such a program, considering that Microsoft discontinued their app, which was not so clear anyway.

The problem is that, visually impaired and/or blind ones like me, can't use the software due to its inaccessibility with NVDA.
You can find more info about NVDA on https://www.nvaccess.org/ - that's the free software most blind people use.

Is integration with NVDA or other screen readers planned?

Thanks.

**Comment by Belim** [comment]
i have not yet dealt with barrier-free development and need to look into this. i know that this is possible by nature with the .NET languages but unfortunately i have no experience with this yet.
